UnityProject/Assets/Scripts/Networking/NetworkUtils.cs

```diff
@@ -0,0 +1,36 @@
+
+using Mirror;
+using UnityEngine;
+
+/// <summary>
+/// Utils for working with networking.
+/// </summary>
+public static class NetworkUtils
+{
+	/// <summary>
+	/// Tries to find the object with the given net ID, returns null if unsuccessful. Logs
+	/// a warning only if the netID is not empty and not invalid but not found.
+	/// </summary>
+	/// <param name="netId"></param>
+	/// <returns></returns>
+	public static GameObject FindObjectOrNull(uint netId)
```

Comment:
Nice and useful method that centralize a very frequent call including a log is a good idea.